### DISCUSSION TITLE
How to export to SVG without "http://www.w3.org/1999/xhtml" namespace?

### DISCUSSION THREAD

**Discussion by ixfd64** [jgraph/drawio#4505] [Q&A]
I'm a Wikipedia editor and sometimes use diagrams.net to create SVGs to upload to Wikimedia Commons. 

SVG files created with the default options are rejected due to the `http://www.w3.org/1999/xhtml` namespace being deemed unsafe, but I was able to resolve this by choosing "Convert labels to SVG" under Text Settings. However, the Text Settings no longer seems to be available. @alderg mentioned in [another post](https://github.com/jgraph/drawio/discussions/4137#discussioncomment-8242663) that this has been replaced by the "Embed Fonts" option, but the `http://www.w3.org/1999/xhtml` namespace is now always present whether or not "Embed Fonts" is checked. Is there another setting I can change?

**Comment by skybitscom** [comment]
I would also consult with w3.org with your request as well because of the errors you receive such as the unsafe one which yes I am familiar with that one. I would also look under (right click) the Site settings in icon on the left of the URL/choose to allow whatever is blocked but only do one at a time and then reload the page and try again.
